<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CConsoleForm</class>
 <widget class="QWidget" name="CConsoleForm">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>486</width>
    <height>418</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>400</width>
    <height>200</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <property name="leftMargin">
    <number>3</number>
   </property>
   <property name="topMargin">
    <number>3</number>
   </property>
   <property name="rightMargin">
    <number>3</number>
   </property>
   <property name="bottomMargin">
    <number>3</number>
   </property>
   <property name="spacing">
    <number>3</number>
   </property>
   <item row="0" column="0">
    <layout class="QHBoxLayout" name="headInputLayout">
     <property name="spacing">
      <number>3</number>
     </property>
     <item>
      <widget class="QToolButton" name="recordToolButton">
       <property name="text">
        <string>start</string>
       </property>
       <property name="icon">
        <iconset resource="canary-img.qrc">
         <normaloff>:/img/start_record</normaloff>:/img/start_record</iconset>
       </property>
       <property name="toolButtonStyle">
        <enum>Qt::ToolButtonTextBesideIcon</enum>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QToolButton" name="clearToolButton">
       <property name="text">
        <string>clear</string>
       </property>
       <property name="icon">
        <iconset resource="canary-img.qrc">
         <normaloff>:/img/edit_clear</normaloff>:/img/edit_clear</iconset>
       </property>
       <property name="toolButtonStyle">
        <enum>Qt::ToolButtonTextBesideIcon</enum>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QLabel" name="label">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="text">
        <string>log level:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="logLevelComboBox">
       <item>
        <property name="text">
         <string notr="true">Verbose</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string notr="true">Debug</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string notr="true">Info</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string notr="true">Warn</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string notr="true">Error</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string notr="true">Assert</string>
        </property>
       </item>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label_4">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="text">
        <string>log format:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="logFormatComboBox">
       <item>
        <property name="text">
         <string>brief</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>process</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>tag</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>thread</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>raw</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>time</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>long</string>
        </property>
       </item>
      </widget>
     </item>
    </layout>
   </item>
   <item row="1" column="0">
    <widget class="QTextEdit" name="logContentTextEdit">
     <property name="contextMenuPolicy">
      <enum>Qt::CustomContextMenu</enum>
     </property>
     <property name="sizeAdjustPolicy">
      <enum>QAbstractScrollArea::AdjustIgnored</enum>
     </property>
     <property name="lineWrapMode">
      <enum>QTextEdit::NoWrap</enum>
     </property>
     <property name="readOnly">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item row="2" column="0">
    <layout class="QHBoxLayout" name="bottomInputLayout">
     <property name="spacing">
      <number>3</number>
     </property>
     <item>
      <widget class="QLabel" name="label_3">
       <property name="text">
        <string>Tag filter:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="filterTagLineEdit">
       <property name="clearButtonEnabled">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label_5">
       <property name="text">
        <string>content filter:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="filterContentLineEdit">
       <property name="clearButtonEnabled">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label_6">
       <property name="text">
        <string>process filter:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="filterPIDLineEdit">
       <property name="clearButtonEnabled">
        <bool>true</bool>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="canary-img.qrc"/>
 </resources>
 <connections/>
</ui>
